aiida/engine/processes/builder.py

```diff
@@ -188,4 +193,5 @@ def __init__(self, process_class):
 
     @property
     def process_class(self):
+        """Straighforward wrapper for the process_class"""
```

Comment:
Seems this docstring got reverted incorrectly

Reply:
Thanks for the catch! I think it should be ready now.